prefetch.js

```diff
@@ -109,16 +107,21 @@
       return true;
     }
 
+    function createLinkTag(url){
+      var link = document.createElement('link');
+      link.setAttribute('rel', 'prefetch');
+      link.setAttribute('href', url);
+      return link;
+    }
+
     function injectPrefetchLink(a){
-      if(a && isPrefetchable(a)){
-        var url = (typeof a === 'object') ? a.href : a;
-        var link = document.createElement('link');
-        link.setAttribute('rel', 'prefetch');
-        link.setAttribute('href', url);
+      var url = (typeof a === 'object') ? a.href : a;
```

Comment:
Typeof `null` is an `object` you need to make sure you are checking if the value is truthy first.

```javascript
(a && typeof a === 'object') ? a.href : a;
```